src/components/story-block.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Media from 'react-bootstrap/Media'
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import Img from 'gatsby-image'
+
+const StoryBlock = ({
+  imageSource,
+  altText,
+  description,
+  imageRight = false,
+}) => (
+  <section>
+    <Container>
+      <Row>
+        <Col sm={{ span: '6', order: `${imageRight ? 'last' : 'first'} ` }}>
```

Comment:
Checkout the 'order' param here: https://react-bootstrap.github.io/layout/grid/#responsive-grids

And here we're using a nice friendly version of the ternary operator. The last shall be first and the first shall be last. Very Biblical.

Reply:
Ok. Is the order param kind of like a media query?

And the ternary operator in this case is the `imageRight` ?

Reply:
Great question! That's basically correct:
![order](https://user-images.githubusercontent.com/10539855/85793094-19d9ba00-b6fa-11ea-8201-21064ebd6e9b.png)
The React JSX code compiles to html, which adds these bootstrap class names, that link to their stylesheet where they've handled most of the media query stuff. 

For the ternary operator: 

```imageRight ? 'last' : 'first'```

That is it. 
The brackets around it with the dollar sign make a template literal: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

So its basically javascript ternary operator inside a javascript expression that is a template literally inside a javascript expression that is the whole thing in double brackets.